adage/genes/tests.py

```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
+# Create your tests here.
```

Comment:
The original `django-genes` has a big `tests.py` file, but I didn't copy it here, because its conversion to Python3 is not trivial. I will add a issue and one of us can work on it later.